### PR TITLE
Update all demo scripts for use on GKE with a k8s 1.31 alpha cluster

### PR DESCRIPTION
**Pull request by klueska**

### CODE REVIEW

File: demo/clusters/gke/create-cluster.sh [hunk 1]
```diff
@@ -35,6 +35,7 @@ DRIVER_NAME=$(from_versions_mk "DRIVER_NAME")
 
 NETWORK_NAME="${DRIVER_NAME}-net"
 CLUSTER_NAME="${DRIVER_NAME}-cluster"
+NODE_VERSION="1.31.1-gke.2105000"
```
Comment (andrewsykim) [inline demo/clusters/gke/create-cluster.sh:38]: I would recommend we don't specify the full version `1.31.1-gke.2105000"`, as `2105000` will not be available forever. Instead just specifying `v1.31.1` will default to the latest 1.31 GKE version

Comment (klueska) [reply demo/clusters/gke/create-cluster.sh]: Ack. Updated.

**Comment by andrewsykim** [comment]
thanks for updating this @klueska -- I plan to give it a test shortly!

**Pull request merged**